chapters/rum/chapter0/1.mdx

```diff
@@ -0,0 +1,110 @@
+# Introducere[[introducere]]
+
+Bun venit la cursul Hugging Face! Această introducere te va ghida în configurarea environment-ului de lucru. Dacă abia începi cursul, îți recomandăm să arunci o privire mai întâi asupra [Capitolului 1](/course/chapter1), apoi să te întorci și să îți configurezi environment-ul pentru a putea încerca singur codul.
+
+Toate bibliotecile pe care le vom folosi în acest curs sunt disponibile ca Python packages, așa că aici îți vom arăta cum să configurezi Python environment-ul și să instalezi librăriile specifice de care ai nevoie.
+
+Vom acoperi două moduri de a-ți configura environment-ul, folosind un notebook Colab sau un Python environment. Simte-te liber să alegi pe cel care ți se potrivește cel mai bine. Pentru începători, recomandăm să începi folosind un notebook Colab.
+
+Reține că nu vom acoperi sistemul Windows. Dacă folosești Windows, îți recomandăm să urmezi pașii folosind un notebook Colab. Dacă folosești o distribuție Linux sau macOS, poți folosi oricare dintre abordările descrise aici.
+
+Majoritatea cursului se bazează pe faptul că ai un cont Hugging Face. Îți recomandăm să creezi unul acum: [crează un cont](https://huggingface.co/join).
+
+## Folosirea unui notebook Google Colab[[folosind-un-notebook-google-colab]]
+
+Folosirea unui notebook Colab este cea mai simplă configurare posibilă; deschide un notebook în browserul tău și începe imediat să scrii cod!
+
+Dacă nu ești familiarizat cu Colab, îți recomandăm să începi urmând [introducerea](https://colab.research.google.com/notebooks/intro.ipynb). Colab îți permite să folosești hardware accelerat, cum ar fi GPU-uri sau TPU-uri, și este gratuit pentru sarcini mai mici.
+
+Odată ce ești confortabil să te descurci în Colab, creează un nou notebook și începe cu configurarea:
+
+<div class="flex justify-center">
+<img src="https://huggingface.co/datasets/huggingface-course/documentation-images/resolve/main/en/chapter0/new_colab.png" alt="Un notebook Colab gol" width="80%"/>
```

Comment:
Nice job translating the alt text as well! 👏

Reply:
Thx a lot! Should the person be from the hugging face team, or it can also be a friend of mine that helps me with the translation?

Reply:
It can be a friend 🤗

Reply:
Hi, I am Ludmila, I am a native Romanian speaker, I had a look through eduard-balamatiuc 's translation process. Everything looks good till now!

Reply:
Awesome, thanks for reviewing!